src/main/java/me/jellysquid/mods/sodium/client/render/chunk/compile/ChunkBuilder.java

```diff
@@ -113,6 +162,8 @@ public void stopWorkers() {
             throw new IllegalStateException("No threads are alive but the executor is in the RUNNING state");
         }
 
+        // Temporary seedqueue compatibility - log less so that its inject works
+        // LOGGER.info("Stopping {} worker threads", this.threads.size());
```

Comment:
SeedQueue suppresses some log messages, which limits our ability to change them. Ideally we would log the worker thread count here. Could just format a string then pass it in.